Formula/avian.rb

```diff
@@ -22,7 +23,25 @@ class Avian < Formula
 
   uses_from_macos "zlib"
 
+  # Fixes:
+  # - CVE-2020-17360
+  # - CVE-2020-17361
+  # - CVE-2020-28371
```

Comment:
Will these be in the next release?

Reply:
There will not be a next release. This formula is deprecated as unmaintained.

Maybe we should just disable it, as I don't know why it crashes on Catalina and Big Sur (probably some obscure compiler change).

Reply:
Unmaintained, can't build and has security issues. I think it ticks all boxes to be disabled.